packages/spaces/lib/vpn-connections.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+module.exports = function (heroku) {
+  function postVPNConnections (space, name, ip, cidrs) {
+    return request('POST', `/spaces/${space}/vpn-connections`, {
+      name: name,
+      public_ip: ip,
+      routable_cidrs: cidrs
+    })
+  }
+
+  function request (method, path, body) {
+    return heroku.request({
+      method: method,
+      path: path,
+      body: body,
+      headers: { Accept: 'application/vnd.heroku+json; version=3.dogwood' }
```

Comment:
Is this variant necessary for the path (`/spaces/${space}/vpn-connections`) ?

Reply:
Done in dd07aa36.